msi/make.cmd

```diff
@@ -0,0 +1,33 @@
+rmdir /s/q build
+
+rem TODO: Where should the git-credential-winstore, gcm and posh-git binaries
+rem       be found/stored?
```

Comment:
I understand what you're saying, and I will try to help you there, but my main interest was to shut up the many users who want .msi installers, but not badly enough to lift even one finger. :smile: